src/main/java/com/project/videotecha/model/User.java

```diff
@@ -11,7 +11,7 @@
 import java.util.Objects;
 
 @Entity
-@Table(name="users")
+@Table(name = "\"user\"")
```

Comment:
Your table name will be called `user` by default. no need to use this annotation